go.mod

```diff
@@ -66,10 +70,14 @@ require (
 )
 
 replace (
+	// FIXME gogo/googleapis must be v1.3.2 <BECAUSE>
 	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
```

Comment:
This replace was added in https://github.com/containerd/containerd/commit/6a2df5fcecf8ada5ce2e1e01f45b5f9ee65f08a0 (https://github.com/containerd/containerd/pull/4760). Most likely, this was only done to keep the vendored versions the same as before switching to go.mod, but we should add an explanation (and check if this is still required)

Reply:
googleapis should be safe, it was kept because bumping will cause some pb.go changed, again IIRC :joy:

Reply:
Thanks for looking! I'll wait for others to comment if they have more details, and will update the PR based on that.